adapters/repos/db/vector/hnsw/condensor.go

```diff
@@ -228,6 +235,27 @@ func (c *MemoryCondensor) AddTombstone(nodeid uint64) error {
 	return ec.ToError()
 }
 
+func (c *MemoryCondensor) AddPQ(data ssdhelpers.PQData) error {
```

Comment:
We have a similar method in weaviate/adapters/repos/db/vector/hnsw/commitlog/logger.go. I do notice they both (Logger and Condensor) have independent buffers (toWrite) but we could still have a helper function receiving the buffers and doing the duplicated work. This way we avoid potential bugs in the future when changing the code. Maybe the comment goes beyond the PR since we also have a similar situation for example for AddNode method (not new in the PR).